docs/assets/2.11-2.12-cleanup-servicemonitors.sh

```diff
@@ -4,6 +4,7 @@ kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring
 kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring.coreos.com eventing-controller
 kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring.coreos.com eventing-nats
 kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring.coreos.com eventing-publisher-proxy
+kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring.coreos.com logging-loki
```

Comment:
Add tracing ServiceMonitor and Loki Rules to cleanup script.

Reply:
Good point